bin/src/commands/launch/mod.rs

```diff
@@ -142,6 +143,25 @@ pub fn execute(matches: &ArgMatches) -> Result<Report, Error> {
 
     trace!("launch config: {:?}", launch);
 
+    // extend matches with config args before continuing
+    let Ok(matches) = cli().try_get_matches_from({
```

Comment:
Maybe try `get_matches_from` here too? It looks like it does pretty much the same thing but with error handling